ginkgo_linter.go

```diff
@@ -371,6 +368,19 @@ func checkAsyncAssertion(pass *analysis.Pass, config types.Config, expr *ast.Cal
 	return true
 }
 
+func isValidFuncCallInAsync(t gotypes.Type) bool {
```

Comment:
I was initially confused about the `FuncCall` here. This isn't checking the *function call*, it's checking the return type of the function call, is it?

Perhaps rename to `isValidAsyncValueType`?